components/expenses/ExpenseItemForm.js

```diff
@@ -125,18 +125,14 @@ const ExpenseItemForm = ({
                   label={<AttachmentLabel />}
                   data-cy="attachment-url-field"
                   required={!isOptional}
-                  error={
-                    meta.error?.type === ERROR.FORM_FIELD_REQUIRED
-                      ? formatMessage(msg.receiptRequired)
-                      : formatFormErrorMessage(intl, meta.error)
-                  }
+                  error={meta.error?.type !== ERROR.FORM_FIELD_REQUIRED && formatFormErrorMessage(intl, meta.error)}
                 >
                   <StyledDropzone
                     {...attachmentDropzoneParams}
                     data-cy={`${field.name}-dropzone`}
                     name={field.name}
                     isMulti={false}
-                    error={meta.error}
+                    error={meta.error?.type === ERROR.FORM_FIELD_REQUIRED && formatMessage(msg.receiptRequired)}
```

Comment:
Won't this set `error={false}` if the error type is something else than `ERROR.FORM_FIELD_REQUIRED`? I haven't tested, but feel like the code should be something like:

```suggestion
                    error={meta.error?.type === ERROR.FORM_FIELD_REQUIRED ? formatMessage(msg.receiptRequired) : meta.error}
```